scripts/flattener.py

```diff
@@ -843,7 +861,7 @@ def reconcile_genes(cxg_adata_lst):
 	for key in stats:
 		stats[key] = set(stats[key])
 		overlap_norm = set(mfinal_adata_genes).intersection(stats[key])
-		logging.info("{}\t{}\t{}\t{}\t{}".format(key, len(stats[key]), len(overlap_norm), overlap_norm, stats[key]))
+		warning_list.append("{}\t{}\t{}\t{}".format(key, len(stats[key]), len(overlap_norm), overlap_norm))
```

Comment:
For line 850: `stats['redundant'] = redundant`, it is adding a list of Ensembl IDs that map to redundant gene symbols. And when I flatten LATDF695OQI, I am getting this in the log:

> redundant	110	0	set()
> WARNING: 8 redundant gene_ids dropped:	['ENSG00000180425', 'ENSG00000256806', 'ENSG00000196139', 'ENSG00000112096', 'ENSG00000105519', 'ENSG00000128655', 'ENSG00000196642', 'ENSG00000150076']

The first line is confusing because the for loop in line 861 is checking for all redundant ID that overlap the gene symbols in mfinal_adata_genes. This will always be 0, since mfinal_adata_genes is gene symbol and the redundant  is Ensembl ID. (Can you confirm this?). I think the first line is confusing and should be removed. So, I would either drop `stats['redundant'] = redundant`, or else the other option is to select for it in the `for key in stats:` loop, and not print the last two elements in the format parameters.

Hopefully that makes sense. Otherwise, looks great!!

Reply:
> > This will always be 0, since mfinal_adata_genes is gene symbol and the redundant is Ensembl ID. (Can you confirm this?).

Can confirm.
I'll go ahead and drop `stats['redundant'] = redundant` then.